servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Main.java

```diff
@@ -149,6 +171,7 @@ private static CodeGeneratorResponse generate(final CodeGeneratorRequest request
         }
         final String typeSuffixValue = optionsMap.get(TYPE_NAME_SUFFIX_OPTION);
         final boolean printJavaDocs = parseBoolean(optionsMap.getOrDefault(PRINT_JAVA_DOCS_OPTION, "true"));
+        final boolean defaultServiceMethods = parseBoolean(optionsMap.getOrDefault(DEFAULT_SERVICE_METHODS, "false"));
```

Comment:
Please add a comment: `// FIXME: 0.43 - consider changing default value to true`